lib/contextMenu.js

```diff
@@ -173,6 +173,33 @@ export default function (self) {
         self.parentDOMNode.appendChild(downArrow);
       }
     }
+    function fade(element) {
+      var op = 1;
```

Comment:
Where possible, please use descriptive variable names — for instance, I think `op -> opacity` is more clear?